parser-typechecker/src/Unison/Codebase/SqliteCodebase/Operations.hs

```diff
@@ -565,11 +565,11 @@ namesAtPath path = do
       allTypes = typesInPath <> typesOutsidePath
   let rootTerms = Rel.fromList allTerms
   let rootTypes = Rel.fromList allTypes
-  let absoluteRootNames = Names {terms = rootTerms, types = rootTypes}
-  let absoluteExternalNames = Names {terms = Rel.fromList termsOutsidePath, types = Rel.fromList typesOutsidePath}
+  let absoluteRootNames = Names.makeAbsolute $ Names {terms = rootTerms, types = rootTypes}
```

Comment:
In a future PR, investigate whether we actually want absoluteRootNames in parseNames at all.